server/handlers/changefeed.go

```diff
@@ -77,9 +77,15 @@ func checkChangefeedInputs(logger ctxu.Logger, s interface{}, r string) (
 			)
 			return
 		}
-		if pageSize == 0 {
-			pageSize = notary.DefaultPageSize
-		}
+
+	}
+	switch {
+	case pageSize == 0:
+		pageSize = notary.DefaultPageSize
+	case pageSize > notary.MaxPageSize:
```

Comment:
Definitely, was planning to. @cyli asked for them while I was on my way out of the office last night :)